pkgs/tools/networking/easyrsa/default.nix

```diff
@@ -1,33 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, openssl, runtimeShell }:
+{ lib, stdenv, fetchFromGitHub, openssl, makeWrapper, runtimeShell }:
 
-let
-  version = "3.0.8";
-in stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "easyrsa";
-  inherit version;
+  version = "3.1.1";
 
   src = fetchFromGitHub {
     owner = "OpenVPN";
     repo = "easy-rsa";
     rev = "v${version}";
-    sha256 = "05q60s343ydh9j6hzj0840qdcq8fkyz06q68yw4pqgqg4w68rbgs";
+    sha256 = "sha256-errF7bNhX3oYEMDwB/B1W5hBWhOD+GCgET3lA121PHc=";
   };
 
-  patches = [ ./fix-paths.patch ];
+  nativeBuildInputs = [ makeWrapper ];
 
   installPhase = ''
-    mkdir -p $out/share/easyrsa
-    cp -r easyrsa3/{*.cnf,x509-types,vars.example} $out/share/easyrsa
-    cp easyrsa3/openssl-easyrsa.cnf $out/share/easyrsa/safessl-easyrsa.cnf
+    mkdir -p $out/share/easy-rsa
+    cp -r easyrsa3/{*.cnf,x509-types,vars.example} $out/share/easy-rsa
     install -D -m755 easyrsa3/easyrsa $out/bin/easyrsa
+
     substituteInPlace $out/bin/easyrsa \
-      --subst-var out \
-      --subst-var-by openssl ${openssl.bin}/bin/openssl
+      --replace /usr/ $out/ \
+      --replace '~VER~' '${version}' \
+      --replace '~GITHEAD~' 'v${version}' \
+      --replace '~DATE~' '1970-01-01'
```

Comment:
I see this pattern a lot across nixpkgs and wonder if a utility function would be useful that turns the current nixpkgs version into a dummy date. 
(For example, 22.11 -> 2022-11-01).

Of course, the downside is that this forces a rebuild when nixpkgs changes, but the date is more "accurate".

May do this in a separate commit because it's come up in a handful of packages I maintain.

Reply:
> (For example, 22.11 -> 2022-11-01).

That date is for reproducibility. It would probably trigger some test failures somewhere when we change that frequently. It is better to just stick to it. Please keep it as is. For zip 1.1.1980 is fine, too.